README.md

```diff
@@ -47,47 +39,17 @@ The ClusterLink APIs use the following entities for configuring cross cluster co
 - Policy. Represent communication rules that must be enforced for all cross-cluster
  communications at each ClusterLink gateway.
 
-## Getting Started
-
-### Building ClusteLink
-
-<!-- We have a [tutorial](TODO missing link) that walks you through setting up your developer
- environment, making a change and testing it.-->
-
-Here are the key steps for setting up your developer environment, making a change and testing it:
+For further information, please refer to the [concepts section](https://clusterlink.net/docs/main/concepts/)) on the ClusterLink website.
```

Comment:
nit: you should likely not refer to 'version` in the URL. If anything, I would recommend the latest released version,,,
But it may be best if we add redirects from /docs/* to /docs/latest/* in netlify.
I still need to figure it out and then configure it...

Reply:
Also, I started using named links and putting all URL at the bottom of the markdown doc. This makes later updates easier as they are not mixed with text.
The format is one of:
- `[link text][link name]`; or
- `[link text][]` (empty brackets imply `link name` == `link text`)

Either should by followed by `[link name]: URL` at the bottom. Link names may contain spaces. You can have multiple links (possibly with different text) referencing the same URL, defined once.

Reply:
For now, I use main until we have the latest redirect